cluster/addons/limit-range/limit-range.json

```diff
@@ -0,0 +1,20 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "LimitRange",
+  "metadata": {
+    "name": "limits"
+  },
+  "spec": {
+    "limits": [
+    {
+      "type": "Container",
+      "min": {
+        "cpu": "400m"
```

Comment:
And is there a reason you're using json rather than yaml?

YAML is generally easier to read, and supports comments -- #7257